coderd/database/queries/workspacescripts.sql

```diff
@@ -1,6 +1,6 @@
 -- name: InsertWorkspaceAgentScripts :many
 INSERT INTO
-	workspace_agent_scripts (workspace_agent_id, created_at, log_source_id, log_path, script, cron, start_blocks_login, run_on_start, run_on_stop, timeout_seconds)
+	workspace_agent_scripts ( workspace_agent_id, created_at, log_source_id, log_path, script, cron, start_blocks_login, run_on_start, run_on_stop, timeout_seconds, display_name, id )
```

Comment:
Unfortunately we don't have a formatter, but the `( ` spacing is uncommon.
```suggestion
	workspace_agent_scripts (workspace_agent_id, created_at, log_source_id, log_path, script, cron, start_blocks_login, run_on_start, run_on_stop, timeout_seconds, display_name, id)
```

Reply:
Oops yeah my bad on this, not sure how the spaces ended up there